File: src/main/java/nu/peg/slack/pt/api/slack/model/AttachmentField.java
package nu.peg.slack.pt.api.slack.model;

import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class AttachmentField {
    private String title, value;
    @SerializedName("short")
    private boolean isShort;
}

File: src/main/kotlin/nu/peg/slack/pt/api/slack/model/AttachmentField.kt
package nu.peg.slack.pt.api.slack.model

import com.google.gson.annotations.SerializedName

import lombok.AllArgsConstructor
import lombok.Data

@Data
@AllArgsConstructor
class AttachmentField {
    private val title: String? = null
    private val value: String? = null
    @SerializedName("short")
    private val isShort: Boolean = false
}
